config/plugins/visualizations/charts/static/repository/utilities/jobs.js

```diff
@@ -43,7 +43,7 @@ define( [ 'utilities/utils' ], function( Utils ) {
                     }
                     chart.state( 'failed', 'This visualization requires the \'' + parameters.tool_id + '\' tool. Please make sure it is installed. ' + message );
                     error && error();
-                          
```

Comment:
Same as above, were these changes intentional?  Might be good to remove unrelated tweaks to avoid rebuilding unnecessary parts of the client.